learningobjects/models.py

```diff
@@ -32,12 +32,12 @@ class LearningObject(models.Model):
     path_xml = models.TextField()
     mpath = models.TextField()
```

Comment:
Last I heard, mptt was not considered necessary, since the individual `LearningObject`'s relationship with the other `LearningObject`s is unimportant once we're past the import stage.

I don't remember hearing about renaming the field, but it's simple, so someone let me know if I should do that.